daemon/mgr/image_load.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	ociimage "github.com/containerd/containerd/images/oci"
 	pkgerrors "github.com/pkg/errors"
+	"github.com/alibaba/pouch/ctrd"
```

Comment:
please follow the package import guidance: https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule003---import-packages .
I think this is not the first time to remind. 
Maybe in the future we could follow the rule much stricter. @wangforthinker 
😄 🐯 🦁 🐤